Release 2.1.1

rocky opened the pull request:
Tentative changes for release 2.1.1

enzo1982 commented:
Thank you for preparing a new release!

This should probably be mentioned in the change log: 6f2426e

rocky commented:
> Thank you for preparing a new release!
> 
> This should probably be mentioned in the change log: [6f2426e](https://github.com/libcdio/libcdio/commit/6f2426e8bf4dc5269ccbd9fbfa94340895f8be6e)

Yes, thanks. I will do a little bit later today. There will be time for people to look over the revised draft.

enzo1982 commented on NEWS.md:
```diff
@@ -3,6 +3,40 @@ NEWS
+- Update freedb references to GnuDB. (Robert Kausch)
+- Fix charset check in Windows cdio_charset_from_utf8 implementation. (Robert Kausch)
+- Add support for reading CD-Text using the Win32 IOCTL driver (Robert Kausch)
+
```
`- Add support for reading CD-Text on OSX`

Should be mentioned here.

rocky replied:
Thanks - fixed now.

enzo1982 commented:
I noticed the OSX CD-Text pull request had been merged here instead of the master branch. I moved it there instead and rebased this PR on the head of the master branch.

rocky commented:
> I noticed the OSX CD-Text pull request had been merged here instead of the master branch. I moved it there instead and rebased this PR on the head of the master branch.

Thanks - if you find more mistakes, please feel free to make additional changes as well.
